schema.sql

```diff
@@ -0,0 +1,44 @@
+CREATE DATABASE `810259-yeticave-11`
+  DEFAULT CHARACTER SET utf8
+  DEFAULT COLLATE utf8_general_ci;
+
+  USE `810259-yeticave-11`;
+
+CREATE TABLE user (
+  id                INT AUTO_INCREMENT PRIMARY KEY NOT NULL,
+  date_add          DATETIME NOT NULL,
+  email             VARCHAR(128) NOT NULL UNIQUE,
+  name              VARCHAR(128) NOT NULL,
+  password          CHAR(64) NOT NULL,
+  contact           TEXT NOT NULL
+);
+
+CREATE TABLE lot (
+  id              INT AUTO_INCREMENT PRIMARY KEY NOT NULL,
+  date_add        DATETIME NOT NULL,
+  name            VARCHAR(128) NOT NULL,
+  description     TEXT NOT NULL,
+  img_url         VARCHAR(128) NOT NULL,
+  first_price     INT NOT NULL,
+  last_price      INT NOT NULL,
+  end_date        DATETIME NOT NULL,
+  bid_step        INT NOT NULL,
+  user_id         INT NOT NULL,
+  categorie_id    INT NOT NULL,
```

Comment:
Если переименуем таблицу ниже, то тут тоже надо
```suggestion
  category_id INT NOT NULL,
```